python/lsst/ap/verify/testPipeline.py

```diff
@@ -455,8 +455,10 @@ def run(self, diaSourceCat):
         results : `lsst.pipe.base.Struct`
             Results struct with components.
         """
+        # TODO Add docstrings for diffIm
```

Comment:
Good idea 😜

Reply:
Added docstrings and updated the results docstrings to be more comprehensive.